addons/vagrant/inventory/hosts

```diff
@@ -0,0 +1,80 @@
+---
+# inventory file use by Vagrantfile
+# to create VMs
+# also use as a Ansible inventory file to provision VMs
+all:
+  children:
+    pfservers:
+      children:
+        stable:
+          hosts:
+            pfcenstable:
+              box: centos/7
+              admin_port: 1543
+              ssh_port: 2300
+              mgmt_ip: 172.17.1.11
+              reg_ip: 192.168.99.110
+              iso_ip: 192.168.100.110
+              ansible_host: 127.0.0.1
+              ansible_port: "{{ ssh_port }}"
+              cpus: 1
+              memory: 4096
+            pfdeb9stable:
+              box: debian/stretch64
+              admin_port: 1943
+              ssh_port: 2700
+              mgmt_ip: 172.17.1.15
+              reg_ip: 192.168.103.110
+              iso_ip: 192.168.104.110
+              ansible_host: 127.0.0.1
+              ansible_port: "{{ ssh_port }}"
+              cpus: 1
+              memory: 8192
+            pfdeb10stable:
+              box: debian/buster64
+              admin_port: 1743
+              ssh_port: 2500
+              mgmt_ip: 172.17.1.13
+              reg_ip: 192.168.101.110
+              iso_ip: 192.168.102.110
+              ansible_host: 127.0.0.1
+              ansible_port: "{{ ssh_port }}"
+              cpus: 1
+              memory: 8192
+        dev:
+          hosts:
+            pfcendev:
+              box: centos/7
+              admin_port: 1443
+              ssh_port: 2200
+              mgmt_ip: 172.17.1.10
+              reg_ip: 192.168.99.10
+              iso_ip: 192.168.100.10
+              ansible_host: 127.0.0.1
+              ansible_port: "{{ ssh_port }}"
+              cpus: 1
+              memory: 4096
+            pfdeb10dev:
+              box: debian/buster64
+              admin_port: 1643
+              ssh_port: 2400
+              mgmt_ip: 172.17.1.12
+              reg_ip: 192.168.101.10
+              iso_ip: 192.168.102.10
+              ansible_host: 127.0.0.1
+              ansible_port: "{{ ssh_port }}"
+              cpus: 1
+              memory: 8192
+            pfdeb9dev:
+              box: debian/stretch64
+              admin_port: 1843
+              ssh_port: 2600
+              mgmt_ip: 172.17.1.14
+              reg_ip: 192.168.103.10
+              iso_ip: 192.168.104.10
+              ansible_host: 127.0.0.1
+              ansible_port: "{{ ssh_port }}"
+              cpus: 1
+              memory: 8192
+  vars:
+    tz: Europe/Paris
```

Comment:
default timezone should either be UTC or America/New_York not Europe/:baguette_bread: